### PR TITLE
perf: improve response buffer handling

### PR DESCRIPTION
**Pull request by tbnritzdoge**
2mb of text
petitio x 96.16 ops/sec ±7.49% (67 runs sampled)
this branch x 268 ops/sec ±1.90% (83 runs sampled)

### CODE REVIEW

File: tests/PetitioRequest.test.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ describe("PetitioRequest", () => {
 		const bodyString2 = qs.stringify(body);
 		const bodyBuffer = Buffer.from(bodyString);
 
-		// eslint-disable-next-line node/no-unsupported-features/node-builtins
+		// eslint-disable-next-line
```
Comment (lveneris) [inline tests/PetitioRequest.test.ts:51]: Why was this changed?

**Review by Nytelife26 — APPROVED** [review]
Looks good, ready for merge.
